__test__/components/__snapshots__/indexPage.test.js.snap

```diff
@@ -25,7 +25,7 @@ Array [
             There is no limit to what you can achieve with a pen.. Write, the world is ready to read.
           </p>
           <button
-            className="index-button sc-htpNat hhMDti"
+            className="index-button sc-htpNat eoyyVM"
```

Comment:
This is the change of class name thing I was talking about on snapshot updates

Reply:
Please let's look into it.

Reply:
ok